templates/archives/stories/debugging.jinja2

```diff
@@ -1,38 +1,60 @@
 {% extends 'archives/story.jinja2' %}
 
 {% block story_title %}
-On the Use of Digital Computers
+"Three Man-Months"
 {% endblock %}
 
+{% block story_credit %}Mingfei Duan{% endblock %}
+
 {% block story_body %}
-    Given the emphasis now placed on the speed of computers and the incredible gains of Moore’s Law, one might presume
-    that the primary issue with the computers of the 1950’s was that they were slow.  But lack of processing power
-    was not necessarily the bottleneck here.  <br />
+    Given the emphasis now placed on the speed of computers and the incredible gains of <a href="https://en.wikipedia.org/wiki/Moore%27s_law">
+    Moore’s Law</a>, one might presume that the primary issue with the computers of the 1950’s was that they were
+    slow.  Computers were already much faster and more reliable than any human being could hope to be.  Lack of
+    processing power was not necessarily the bottleneck here.  <br />
+    <figure class="figure float-right" style="height:25%;width:25%">
+        <img src="{{ static('img/stories/debugging.png') }}"
+            class="figure-img img-fluid rounded"
+            alt="Part of source for the Fortran II compiler for IBM 704." />
+        <figcaption class="figure-caption">Part of source for the Fortran II compiler.</figcaption>
+    </figure>
     <br />
     One major problem that Morse identified was the input and output.  Computers back then didn’t have a screen and a
```

Comment:
Non-computer scientists would not know what input and output means, I would try to be a little more descriptive here to explain what those words mean. You can combine these two sentences:

`One major problem that Morse identified was the input and output, as computers of the time didn’t have a screen and a keyboard.`

Reply:
Something else you may want to add here is that it is a user experience issue, which is something we commonly think about today — how humans and computers interact.